### ISSUE TITLE
Tag applications which interact with LLMs.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [ximion/appstream#652]
Paraphrased, as https://github.com/flathub-infra/website/issues/2869#issue-2229263979 requests:

<blockQuote>

I think Flathub should implement an official way for apps to indicate if they're using at least online AI/LLM features, and require all apps using such to disclose which type. This should be displayed prominently, like the verified/unverified badge or the "potentially unsafe" or "community-built" thingies.

I don't want to install and run the app before finding I can't use it because it's sending my data to the most evil corps (IMHO) in the world.

Apps using this offline should possibly also be marked. It's kinda like the "why have an unverified badge" thing: If they mention their AI features in their description or has a name hinting about it, or you've heard rumours from elsewhere, but there's no official info about whether it's online or offline, then that could create uncertainty. And some might not want to use offline models either.

</blockQuote>

**Comment by ximion** [comment]
The described problem isn't limited to AI though - it basically affects any app that does require some kind of online subscription to work.

Labelling apps that needs any kind of subscription may be what we would actually want, rather than singling out remote-AI specifically...

**Comment by RokeJulianLockhart** [comment]
> https://github.com/ximion/appstream/issues/652#issuecomment-2817329228

@ximion, indeed, although I think that telemetry is the more broad category. Internet connection for a subscription, of all things, isn't generally considered telemetry.

**Comment by razzeee** [comment]
OARS already covers both of those usecases

**Comment by razzeee** [comment]
I think it's mostly documented via the page at https://hughsie.github.io/oars/ and the generator.

The wording for in app purchases, might need some extension 
https://github.com/hughsie/oars/blob/bae1896c320fa22de3a7cd131eb0e03444216e5d/docs/generate.html#L537

For diagnostics
https://github.com/hughsie/oars/blob/bae1896c320fa22de3a7cd131eb0e03444216e5d/docs/generate.html#L537

**Comment by forteller** [comment]
> OARS already covers both of those usecases

So marking apps that use AI is already possible? The reason this issue was opened was because I opened an issue about it on Flathub, and they said it needed to be implemented here first. But then I should go back to Flathub and tell them it's already implemented here?

> Labelling apps that needs any kind of subscription may be what we would actually want, rather than singling out remote-AI specifically...

The reason I first proposed this to Flathub was not about subscriptions, but to help those who don't want to use unethical technology* and get their own data they input used to develop it further. So labeling subscription-based apps sounds good too, but it's not what I want with this issue.

(* based on other peoples IP, hammers servers and skyrockets peoples electricity bills, consumes tons of electricity and water, will mostly be used to spread lies and propaganda and destroy people ability to separate lies from truth)

**Comment by RokeJulianLockhart** [comment]
> https://github.com/ximion/appstream/issues/652#issuecomment-2848164518

@forteller, were you thinking of local generative AI?

**Comment by RokeJulianLockhart** [comment]
> https://github.com/ximion/appstream/issues/652#issuecomment-2848212887

In which case, what, exactly, is the problem? I ask because had it been telemetry, as aforestated, apparently OARS designations encompass that. However, local telemetry doesn't exist as a concept outside of user-focussed applications like ActivityWatch. Consequently, you've evidently a criticism of generative AI implementations themselves. In that case, what is the attribute, or intrinsic collection thereof, which is taggable?
